service/src/test/java/bio/terra/workspace/app/controller/WorkspaceApiControllerTest.java

```diff
@@ -282,6 +303,76 @@ public void cloneWorkspace() throws Exception {
         sourceWorkspace.getDisplayName() + " (Copy)", destinationWorkspace.getDisplayName());
   }
 
+  @Test
+  public void cloneRawlsWorkspace() throws Exception {
```

Comment:
Now that this test class is working with snapshots, can it use `BaseUnitTestMockDataRepoService` instead of `BaseUnitTest`? As-is, I think this would call real TDR if someone called `checkReferenceAccess` on these snapshots

Reply:
Good point. That would be a surprise if someone added such a test.